lib/kinesis.test.js

```diff
@@ -4,12 +4,19 @@ const lock = require('./kinesis-lock')
 const RedisBackup = require('./redis-backup')
 
 describe('kinesis', () => {
-
   let redis, overall, segment
   beforeEach(() => {
     redis = new RedisBackup(process.env.REDIS_URL)
-    overall = {le: '1234', digest: '5678', time: 9, bytes: 10, seconds: 12, percent: 0.4}
-    segment = {...overall, segment: 2}
+    overall = {
+      le: '1234',
+      digest: '5678',
+      time: 9,
+      bytes: 10,
+      seconds: 12,
+      percent: 0.4,
+      percentAds: 0.1,
+    }
+    segment = { ...overall, segment: 2, segmentPosition: 1, percentAds: 0.1 }
```

Comment:
This file got linted. But the only changes I made were:

- Adding the new percentAds/segmentPosition keys to these test objects
- Adding the new fields to the `expect.toEqual()` in a couple places

More in-depth testing occurs in `index.test.js` and `arrangement.test.js`.